e2e-tests/cypress/e2e/pages/your-business/contact/yourContact.js

```diff
@@ -0,0 +1,24 @@
+import { FIELD_IDS } from '../../../../../constants/field-ids';
+
+const {
+  CONTACT: {
+    COMPANY_NAME,
+  },
+} = FIELD_IDS.INSURANCE.EXPORTER_BUSINESS;
+
+const yourContactPage = {
+  [COMPANY_NAME]: {
```

Comment:
could this use the `field` function below? Could add hint to the field and then have `details` specific to this field?